tools/src/faircv/Main.hs

```diff
@@ -51,11 +51,11 @@ main = do
         contentsHash   <- generate arbitrary
         assignmentHash <- generate arbitrary
 
-        sub <- return Submission
-            { _sStudentId      = addrS
-            , _sContentsHash   = contentsHash
-            , _sAssignmentHash = assignmentHash
-            }
+        let sub = Submission
+                  { _sStudentId      = addrS
+                  , _sContentsHash   = contentsHash
+                  , _sAssignmentHash = assignmentHash
+                  }
```

Comment:
This is pretty subjective change IMO.
I'm solidary with @Heimdell that `<- pure` is better than `let =` when that decreases indentation of a big code block.

Reply:
But why introduce `pure` together with surrounding `bind`s (which are there implicitly) just because the indentation seems a bit nice? I think that clear distinction between monadic and pure actions in the code is more important than nicer-looking indents.